src/test/java/net/openhft/chronicle/testframework/process/ProcessRunnerTest.java

```diff
@@ -6,6 +6,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@SuppressWarnings("deprecation")
```

Comment:
I'd like to leave the tests in place until we remove this, so we'll just suppress the warnings here